connexion/apis/aiohttp_api.py

```diff
@@ -33,7 +26,7 @@ def oauth_problem_middleware(request, handler):
     except (OAuthProblem, OAuthScopeProblem) as oauth_error:
         return web.Response(
             status=oauth_error.code,
-            body=json.dumps(oauth_error.description).encode(),
+            body=AioHttpApi.jsonifier.dumps(oauth_error.description).encode(),
```

Comment:
oauth_problem_middleware has been replaced in master. Now all of the Exception handling uses `connexion.problem.problem()` to generate the response. `problem()` returns a ConnexionResponse. That then goes through `AioHttpApi.get_response()` -> `AioHttpApi._get_aiohttp_response_from_connexion()` -> `AioHttpApi._cast_body()` where it gets converted to JSON if needed.

So, this change is no longer needed as it is covered by the changes below in this file.